assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -559,6 +561,54 @@ export const DEFAULT_NOTIFICATIONS = {
 			);
 		},
 	},
+	'web-data-stream-not-available-notification': {
+		Component: WebDataStreamNotAvailableNotification,
+		priority: 10,
+		areaSlug: NOTIFICATION_AREAS.BANNERS_ABOVE_NAV,
+		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+		isDismissible: false,
+		checkRequirements: async ( { select, resolveSelect }, viewContext ) => {
+			const viewOnly =
+				SITE_KIT_VIEW_ONLY_CONTEXTS.includes( viewContext );
+
```

Comment:
Would be nice to follow the pattern of commenting why we are resolving random selectors.
```suggestion
// The isModuleConnected() selector relies on the resolution
// of the getModules() resolver.
```